src/tests/end-to-end/tests/details-view/color-contrast.test.ts

```diff
@@ -42,6 +42,9 @@ describe('Color Contrast Violations', () => {
         await detailsViewPage.waitForSelector(detailsViewSelectors.automatedChecksResultSection, {
```

Comment:
Could we replace this check with the new one? If `ruleDetail` is inside `automatedChecksResultSection` I think it should work fine with just the one.

Reply:
Oh! I'll give it a shot :)